vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionSpaceOrderUtility.java

```diff
@@ -0,0 +1,75 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import edu.asu.diging.vspace.core.model.IExhibition;
+import edu.asu.diging.vspace.core.model.IExhibitionSpaceOrderUtility;
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.model.SpacesCustomOrder;
+import edu.asu.diging.vspace.core.model.impl.SpaceStatus;
+import edu.asu.diging.vspace.core.services.IExhibitionManager;
+
+@Component
+public class ExhibitionSpaceOrderUtility implements IExhibitionSpaceOrderUtility {
```

Comment:
that's not really only relevant to controllers but more business logic, so it should go in the core package.